test/test_individual.py

```diff
@@ -3,21 +3,46 @@
 import pytest
 
 import cgp
-from cgp.individual import IndividualSingleGenome
+from cgp import Genome, IndividualSingleGenome, IndividualMultiGenome
```

Comment:
why this `from` import? couldn't use those classes with the `cgp.` prefix as well?

Reply:
yes, it just saves characters in the code.